linux_sql/scripts/host_usage.sh

```diff
@@ -15,29 +16,18 @@ psql_user=$4
 psql_password=$5
 
 curtime=$(date +%Y-%m-%d' '%T)
-
 export PGPASSWORD=$psql_password
 hostname=`hostname -f`
-#echo hostname is $hostname
-#select_host_id_stmt='SELECT host_id FROM host_info WHERE hostname='$hostname''
 host_id=`psql -h $psql_host -p $psql_port -U $psql_user -d $db_name -t -c "SELECT id FROM PUBLIC.host_info WHERE hostname='$hostname';"`
```

Comment:
please combine this query with the `INSERT` statement as a sub query

e.g. 
INSERT INTO PUBLIC.host_usage (...) VALUES (**CHANGE_THIS_TO_A_SUB_QUERY**, $memory_free, $cpu_idle, $cpu_kernel, $disk_io, $disk_available, '$curtime');"